doc/03-component-interfaces/03-component-interfaces.md

```diff
@@ -132,7 +130,45 @@ hierarchy diagram using whatever tools you would like. Then, include a picture
 of it in this folder. You may also embed it just below using markdown syntax
 (i.e., `![ALT TEXT](path/to/file)`).
 
-<!-- TODO: make a diagram of your component hierarchy then delete this comment -->
+classDiagram
```

Comment:
Is this LaTeX? I actually don't know what the resulting diagram looks like, but this is really nicely done.